hooks/terraform_providers_lock.sh

```diff
@@ -15,6 +15,8 @@ function main {
   common::parse_and_export_env_vars
   # JFYI: suppress color for `terraform providers lock` is N/A`
 
+  # Cleanup in case lockfile was not cleaned in previous run of common::terraform_init function
+  rmdir "$PARALLELISM_FALLBACK_LOCK_DIR"
```

Comment:
Maybe check if it exists to avoid `No such file or directory` error?
```suggestion
  [[ -d "$PARALLELISM_FALLBACK_LOCK_DIR" ]] && rmdir "$PARALLELISM_FALLBACK_LOCK_DIR"
```